BridgeAppSDK/SBARoundedButton.swift

```diff
@@ -53,14 +53,18 @@ import UIKit
     
     override open var isEnabled: Bool {
         didSet {
-            // simple way to show disabled state
+            // show as disabled by lowering opacity unless alpha is used to set hidden
```

Comment:
where is alpha used to set hidden? might be worth mentioning here for future reference